.vib/reportserver/goss/reportserver.yaml

```diff
@@ -0,0 +1,22 @@
+command:
+  check-version:
+    exec: cat /opt/bitnami/reportserver/WEB-INF/classes/rsversion.properties
+    exit-status: 0
+    stdout:
+      - {{ .Env.APP_VERSION }}
+user:
+  tomcat:
+    exists: true
+group:
+  root:
+    exists: true
+file:
+  /opt/bitnami/reportserver:
+    filetype: directory
+    exists: true
+    owner: tomcat
+    group: root
```

Comment:
We are actually not checking the owner/group owners normally, though I agree it doesn't hurt to do it here. That said, maybe we should move the permissions check here as well.

In the future we may add the user/group checks to the `check-files/dirs` goss templates but for now we can put it here as it is.

Reply:
I've removed it for now